blocks/ai-chat/src/complete-chat/chat-textfield.tsx

```diff
@@ -19,98 +12,31 @@ export const ChatTextField: FunctionComponent<{
-      submitMessageContent(inputValue);
-    },
-    [inputValue, submitMessageContent],
-  );
+  const handleSubmit = useCallback(() => {
+    setInputValue("");
+    submitMessageContent(inputValue);
+  }, [inputValue, submitMessageContent]);
 
   return (
-    <Box
-      component="form"
+    <BlockPromptInput
+      value={inputValue}
       onSubmit={handleSubmit}
-      sx={{
-        width: "100%",
-        marginBottom: 0,
-      }}
-    >
-      <TextField
-        autoFocus
-        multiline
-        fullWidth
-        onFocus={onFocus}
-        onBlur={onBlur}
-        value={inputValue}
-        onChange={({ target }) => setInputValue(target.value)}
-        onKeyDown={(event) => {
-          const { shiftKey, code } = event;
-          if (!shiftKey && code === "Enter") {
-            handleSubmit(event);
-          }
-        }}
-        placeholder={
-          chatHasStarted ? "Enter a response" : "Begin a conversation"
-        }
-        required
-        disabled={loading}
-        sx={({ palette }) => ({
-          width: 1,
-          [`& .${inputBaseClasses.input}`]: {
-            minHeight: "unset",
-            fontSize: 16,
-            lineHeight: "21px",
-            paddingY: 2.125,
-            paddingLeft: 2.75,
-            paddingRight: 0,
-          },
-          [`& .${inputBaseClasses.disabled}`]: {
-            background: palette.gray[10],
-            color: palette.gray[70],
-          },
-          [`& .${outlinedInputClasses.notchedOutline}`]: {
-            border: `1px solid ${palette.gray[20]}`,
-          },
-        })}
-        InputProps={{
-          endAdornment: (
-            <Button
-              type="submit"
-              variant="tertiary_quiet"
-              disabled={loading}
-              sx={({ palette }) => ({
-                alignSelf: "flex-end",
-                fontSize: 13,
-                fontWeight: 700,
-                letterSpacing: "-0.02em",
-                lineHeight: 1,
-                color: palette.blue[70],
-                textTransform: "uppercase",
-                height: 55,
-                width: 1,
-                maxHeight: 55,
-                maxWidth: 120,
-                minHeight: 51,
-                whiteSpace: "nowrap",
-                [`&.${buttonBaseClasses.disabled}`]: {
-                  color: palette.common.black,
-                  background: "none",
-                },
-              })}
-            >
-              Submit{" "}
-              <ArrowTurnDownLeftIcon
-                sx={{
-                  ml: 1,
-                  fontSize: 12,
-                }}
-              />
-            </Button>
-          ),
-        }}
-      />
-    </Box>
+      onFocus={onFocus}
+      onBlur={onBlur}
+      onChange={({ target }) => setInputValue(target.value)}
+      placeholder={chatHasStarted ? "Enter a response" : "Begin a conversation"}
+      disabled={loading}
+      buttonLabel={
+        <>
+          Submit Prompt{" "}
```

Comment:
This is introducing an inconsistency with the AI Chat block designs, the button should just say "Submit":
```suggestion
          Submit
```